CRITICAL - 2020-11-10 18:09:52 --> syntax error, unexpected '}'
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(869): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(394): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-10 18:10:14 --> Call to a member function getBatch() on null
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Catering_buyer->save()
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Catering_buyer))
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2020-11-10 18:12:19 --> Cannot use object of type stdClass as array
#0 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Catering_buyer->save()
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Catering_buyer))
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2020-11-10 18:17:00 --> Call to undefined method App\Models\Admin\Catering_order_batch_model::getResult
#0 C:\xampp\htdocs\internal-nutrilo\app\Models\Admin\Catering_order_batch_model.php(22): CodeIgniter\Model->__call('getResult', Array)
#1 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Catering_buyer.php(188): App\Models\Admin\Catering_order_batch_model->getBatchBetween('batch_number BE...')
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Catering_buyer->save()
#3 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Catering_buyer))
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2020-11-10 22:50:05 --> Invalid argument supplied for foreach()
#0 C:\xampp\htdocs\internal-nutrilo\app\Views\Admin\Driver_view.php(161): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'C:\\xampp\\htdocs...', 161, Array)
#1 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('Admin/Driver_vi...', Array, true)
#3 C:\xampp\htdocs\internal-nutrilo\app\Controllers\Admin\Driver.php(230): view('Admin/Driver_vi...', Array)
#4 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Admin\Driver->setDriver()
#5 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Driver))
#6 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\internal-nutrilo\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\internal-nutrilo\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
